local_node.sh

```diff
@@ -85,6 +85,7 @@ if [[ $overwrite == "y" || $overwrite == "Y" ]]; then
 	jq '.consensus_params["block"]["max_gas"]="10000000"' "$GENESIS" >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
 
 	# x/assets
+	jq '.app_state["assets"]["params"]["exocore_lz_app_address"]="0x3e108c058e8066da635321dc3018294ca82ddedf"' "$GENESIS" >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
```

Comment:
Is this the CREATE2 address?

Reply:
No, it's the default address from the above local mnemonic. It is used to start the local node, then directly use this EOA address to test all precompiled contract interfaces, without relying on the client chain and Exocore gateway contract.

Reply:
Ah okay - it is the `local_funded_account`. Can you please add a comment for future reference?

Reply:
Yeah